unlock-app/src/components/content/event/Layout/PastEvent/index.tsx

```diff
@@ -173,6 +174,27 @@ export const PastEvent = ({
   }
   return (
     <Card className="grid gap-4 mt-5 md:mt-0">
+      {Object.keys(checkoutConfig.config.locks)
+        ?.sort((l, m) => {
+          return (
+            (checkoutConfig.config.locks[l].order || 0) -
+            (checkoutConfig.config.locks[m].order || 0)
+          )
+        })
+        ?.map((lockAddress: string) => {
+          return (
+            <LockExplorerLink
```

Comment:
Let's actually make this a component that can receive multiple contracts and handle that _inside_ of it.

`<LockExplorerLink checkoutConfig={checkoutConfig}>`